samples/test-vulnerable-20260130.py

```
@@ -0,0 +1,57 @@
+# 2. SQL Injection (HIGH)
+def get_user(user_id):
+    query = "SELECT * FROM users WHERE id = '" + user_id + "'"  # sqli
+    return query
+
+# 3. Hardcoded Password (HIGH)
+DB_PASSWORD = "super_secret_password_123"
+API_KEY = "sk-test-abcdef123456789"
+
+# 4. Insecure Deserialization (CRITICAL)
+def load_data(data):
+    return pickle.loads(data)  # insecure deserialization
```

Ruff (0.14.14)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)